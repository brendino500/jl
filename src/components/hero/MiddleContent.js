import { makeStyles } from '@material-ui/core/styles'
import { Typography, Button } from '@material-ui/core'

import CountdownTimer from './CountdownTimer'
import divider from '../../assets/divider.png'
import { dateString, dateValidation } from '../../config/dateChange'

const useStyles = makeStyles((theme) => ({
  container: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
  },
  timerLayoutMain: {
    display: 'flex',
  },
  divider: {
    width: 'auto',
  },
}))

export default function MiddleContent() {
  const classes = useStyles()
  dateValidation()

  return (
    <div className={classes.container}>
      <CountdownTimer />
      <img src={divider} alt="divider" className={classes.divider} />
      <Typography>Reveal Event {dateString}</Typography>
      <Button
        href="https://square-enix-games.com/"
        target="_blank"
        rel="noreferrer"
      >
        Watch Teaser
      </Button>
    </div>
  )
}
