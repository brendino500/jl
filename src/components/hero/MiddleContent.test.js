import renderer from 'react-test-renderer'
import MiddleContent from './MiddleContent'

it('MiddleContent matches snapshot', () => {
  const tree = renderer.create(<MiddleContent />).toJSON()
  expect(tree).toMatchSnapshot()
})
