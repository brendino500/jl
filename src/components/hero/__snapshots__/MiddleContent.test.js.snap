// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiddleContent matches snapshot 1`] = `
<div
  className="makeStyles-container-1"
>
  <div
    className="makeStyles-container-6"
  >
    <div
      className="makeStyles-timerLayoutMain-8"
    >
      <div
        className="makeStyles-timerFormat-7"
      >
        <div
          className="makeStyles-timeTextLayout-11"
        >
          <p
            className="MuiTypography-root makeStyles-timeNumbers-12 MuiTypography-body1"
          >
            00
          </p>
          <p
            className="MuiTypography-root makeStyles-timeText-13 MuiTypography-body1"
          >
            days
          </p>
        </div>
        <p
          className="MuiTypography-root makeStyles-colon-10 MuiTypography-body1"
        >
          :
        </p>
      </div>
      <div
        className="makeStyles-timerFormat-7"
      >
        <div
          className="makeStyles-timeTextLayout-11"
        >
          <p
            className="MuiTypography-root makeStyles-timeNumbers-12 MuiTypography-body1"
          >
            00
          </p>
          <p
            className="MuiTypography-root makeStyles-timeText-13 MuiTypography-body1"
          >
            hours
          </p>
        </div>
        <p
          className="MuiTypography-root makeStyles-colon-10 MuiTypography-body1"
        >
          :
        </p>
      </div>
      <div
        className="makeStyles-timerFormat-7"
      >
        <div
          className="makeStyles-timeTextLayout-11"
        >
          <p
            className="MuiTypography-root makeStyles-timeNumbers-12 MuiTypography-body1"
          >
            00
          </p>
          <p
            className="MuiTypography-root makeStyles-timeText-13 MuiTypography-body1"
          >
            mins
          </p>
        </div>
        <p
          className="MuiTypography-root makeStyles-colon-10 MuiTypography-body1"
        >
          :
        </p>
      </div>
      <div
        className="makeStyles-timerFormat-7"
      >
        <div
          className="makeStyles-timeTextLayout-11"
        >
          <p
            className="MuiTypography-root makeStyles-timeNumbers-12 MuiTypography-body1"
          >
            00
          </p>
          <p
            className="MuiTypography-root makeStyles-timeText-13 MuiTypography-body1"
          >
            secs
          </p>
        </div>
      </div>
    </div>
  </div>
  <img
    alt="divider"
    className="makeStyles-divider-3"
    src="divider.png"
  />
  <p
    className="MuiTypography-root makeStyles-revealText-4 MuiTypography-body1"
  >
    Reveal Event 
    16th February
  </p>
  <a
    aria-disabled={false}
    className="MuiButtonBase-root MuiButton-root MuiButton-text"
    href="https://square-enix-games.com/"
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    rel="noreferrer"
    tabIndex={0}
    target="_blank"
  >
    <span
      className="MuiButton-label"
    >
      <img
        alt="watch teaser button"
        className="makeStyles-teaserButton-5"
        src="watch-teaser-button.png"
      />
    </span>
  </a>
</div>
`;
